src/SourceBuild/tarball/content/prep.sh

```diff
@@ -31,6 +33,14 @@ while :; do
         --bootstrap)
             buildBootstrap=true
             ;;
+        -runtimesourcefeed)
```

Comment:
To follow the existing convention, these two new options (named with complete words) should use a `--` prefix.

Reply:
They should also be documented.